app/src/main/java/edu/uc/groupProject/topten/ui/main/MainFragment.kt

```diff
@@ -124,19 +116,19 @@ class MainFragment : Fragment() {
         adapter = CurrentListAdapter(viewModel, testList)
         recyclerView.adapter = adapter
 
+        var itemList = viewModel.getItemList()
 
-
-        viewModel.firestoreService.list.observe(this, Observer {
+        itemList.observe(this, Observer {
 
             activity?.runOnUiThread(
 
                 Runnable {
                     val recyclerViewState: Parcelable? =
                         recyclerView.layoutManager!!.onSaveInstanceState()
 
-                    adapter.setItemList(viewModel.firestoreService.list.value!!)
```

Comment:
I changed this to 'it', as that is exactly what is being passed as 'it' inside the observe block.

Reply:
That is a bit more clear.